cpp/include/cudf/lists/detail/sorting.cuh

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
This does not need to be a `.cuh` file since it does not contain any device code.
You can change it to `sorting.hpp`.